mne/preprocessing/maxwell.py

```diff
@@ -5,35 +6,54 @@
 
 # License: BSD (3-clause)
 
-from __future__ import division
+from copy import deepcopy
 import numpy as np
 from scipy import linalg
 from math import factorial
+from os import path as op
 import inspect
 
-from .. import pick_types
+from .. import __version__
+from ..transforms import _str_to_frame, _get_trans
 from ..forward._compute_forward import _concatenate_coils
 from ..forward._make_forward import _prep_meg_channels
+from ..surface import _normalize_vectors
+from ..io.constants import FIFF
+from ..io.open import fiff_open
+from ..io.tree import dir_tree_find
 from ..io.write import _generate_meas_id, _date_now
+from ..io.tag import find_tag, _loc_to_coil_trans
+from ..io.pick import pick_types, pick_info, pick_channels
 from ..utils import verbose, logger
+from ..externals.six import string_types
+from ..channels.channels import _get_T1T2_mag_inds
 
 
 @verbose
-def maxwell_filter(raw, origin=(0, 0, 40), int_order=8, ext_order=3,
-                   st_dur=None, st_corr=0.98, verbose=None):
+def maxwell_filter(raw, origin='default', int_order=8, ext_order=3,
+                   fine_cal=None, ctc=None, st_dur=None, st_corr=0.98,
```

Comment:
There are arguments for both ways I guess. Do we want the params to be clearer at first glance, but less like Elekta's naming and have long variable names to type out or vice versa?